DataTypes.php

```diff
@@ -9,12 +9,12 @@
 
 namespace DataTypes;
 
-if ( defined( 'DataTypes_VERSION' ) ) {
+if ( defined( 'DATATYPES_VERSION' ) ) {
 	// Do not initialize more than once.
 	return 1;
 }
 
-define( 'DataTypes_VERSION', '1.0.0' );
+define( 'DATATYPES_VERSION', '1.0.0' );
```

Comment:
Uh, what's going on here? This check fails now, but Travis doesn't report it: https://travis-ci.org/wmde/DataTypes/jobs/259698637